utils/docker/Dockerfile.centos.7

```diff
@@ -12,13 +12,69 @@ LABEL maintainer="daos@daos.groups.io"
 # Intermittent cache-bust.  Used to reduce load on the actual CACHEBUST later.
 ARG CB0
 
+# Switch to using DNF instead of yum.
+#
+# If a local distro repository is supplied, then we should use it
+# and disable using the distro repositories.
+# The Docker image starts out with the DISTRO provided GPG keys
+# on the disk, but not installed.  These must be isntalled before
+# adding any repo that needs GPG keys but does not provide meta
+# data for installing those keys.  Then the epel-release package
+# version from the extras repository must be installed and the
+# GPG key it provides must be installed.
+ARG REPO_URL=""
+ARG REPO_DISTRO=""
+RUN if [ -n "$REPO_DISTRO" ]; then                              \
```

Comment:
I originally read this as if REPO_URL wasn't set this whole block would be skipeed, but I see there's an else on L45 and some code after the fi.